libs/libc/unistd/lib_sysconf.c

```diff
@@ -229,7 +229,7 @@ long sysconf(int name)
 #ifdef CONFIG_MM_PGSIZE
         return CONFIG_MM_PGSIZE;
 #else
-        return 1;
+        return 4096;
```

Comment:
Please define something like DEFAULT_MM_PGSIZE and return it.

Reply:
Done.